packages/web/src/new/scenes/Main/Receive/ReceiveToken/Bitcoin/Hint/Hint.tsx

```diff
@@ -28,18 +37,14 @@ const Row = styled.li`
 `;
 
 const MinimumTxAmount = styled.div`
-  display: flex;
-
-  &.inline {
-    display: inline;
-  }
+  display: inline-flex;
 `;
 
 interface Props {
-  remainingTime: string;
+  viewModel: ReceiveBitcoinViewModel;
```

Comment:
Don't forget to write Readonly, its just better typing to restrict access to direct change of view model properties